accounting/src/app/page.tsx

```diff
@@ -0,0 +1,12 @@
+import './home.css';
```

Comment:
Could we match the file names here, so that a page and it's specific CSS (if it is specific) match?

So in this case, home.css -> page.css OR page.tsx -> home.tsx

Reply:
On it!

Reply:
Ah, nice then I know! ^^